OpenDreamRuntime/Procs/DMProc.cs

```diff
@@ -868,6 +868,8 @@ public sealed class DMProcState : ProcState {
                     ThrowInvalidAppearanceVar(field);
 
                 return Proc.AtomManager.GetAppearanceVar(appearance, field);
+            } else if (owner.TryGetValueAsType(out var ownerType) && ownerType != null && ownerType.ObjectDefinition.Variables.TryGetValue(field, out var val)) {
```

Comment:
`ownerType` will not be null if `TryGetValueAsType()` succeeds
```suggestion
            } else if (owner.TryGetValueAsType(out var ownerType) && ownerType.ObjectDefinition.Variables.TryGetValue(field, out var val)) {
```